pkg/controllers/vdb/config.go

```diff
@@ -582,3 +582,12 @@ func genUnsupportedVerticaVersionEventMsg(feature, supportedVersion string) stri
 	prefix := "The engine (%s) doesn't have the required change to setup"
 	return fmt.Sprintf("%s %s. You must be on version %s or greater", prefix, feature, supportedVersion)
 }
+
+// getEncryptSpreadComm will return "vertica" if encryptSpreadComm is set to
+// an empty string, otherwise return the value of encryptSpreadComm
+func (g *ConfigParamsGenerator) getEncryptSpreadComm() string {
```

Comment:
Can we move this to VeritcaDB? And rename it so that it's exported (i.e. `GetEncryptSpreadComm`). There is a file (helpers.go) that has these kind of functions.